subtests/docker_cli/systemd/systemd_build.py

```diff
@@ -0,0 +1,28 @@
+"""
+Build docker image using Dockerfile and systemd unit file -
+``docker-build.service`` is used. This unit file is copied to system and
+minor edition is made. A systemd service is started using this file to build
+the image.
+"""
+
+from systemd import systemd_base
+from dockertest.images import DockerImages
+
+
+class systemd_build(systemd_base):
+
+    def initialize(self):
+        super(systemd_build, self).initialize()
+        DockerImages(self).clean_all([self.config['image_name']])
+        var_to_edit = {'NAME=': self.config['image_name'],
+                       'DOCKERFILEDIR=': self.sub_stuff['dirname']}
+        for k, v in var_to_edit.items():
+            self.init_unitfile(k, v)
+
+    def run_once(self):
+        self.sysd_action('daemon-reload')
+        self.sysd_action('start', self.config['sysd_unit_file'])
+
+    def cleanup(self):
+        super(systemd_build, self).cleanup()
+        DockerImages(self).clean_all([self.config['image_name']])
```

Comment:
And use something like image_name for all three in the ini file instead of name, image_name, and image_to_pull?